clientv3/cluster.go

```diff
@@ -33,6 +34,9 @@ type Cluster interface {
 	// List lists the current cluster membership.
 	MemberList(ctx context.Context) (*MemberListResponse, error)
 
+	// Leader returns the current leader member.
```

Comment:
I missed this comment. You mean to make the comment `MemberLeader`? I can change that in a separate PR with the fix for `MemberList` as well.